### PR TITLE
Sprint 13 requirements done.

### PR DESCRIPTION
**Pull request by nnkmk**
- Successfully tested edge cases using Charles Proxy. 
- Refactored WebViewViewController. 
- Covered ProfileViewController and ImageListViewController with Unit tests. 
- Written UI tests to verify app authentication, feed interaction, and profile functionality.

### CODE REVIEW

File: ImageFeed/Auth/AuthConfiguration.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+let fileAccessKey = "caDgYQ_VA-HHrN1zPzRSjt3wosQR2CjsxrJrm-GJnQw"
+let fileSecretKey = "szvvMLYptwe8EwGdheMZUrDbPwW5qKVsP8d49hndOnI"
+let fileRedirectURI = "urn:ietf:wg:oauth:2.0:oob"
+let fileAccessScope = "public+read_user+write_likes"
```
Comment (DanilaG) [inline ImageFeed/Auth/AuthConfiguration.swift:6]: Сейчас, к одним и тем же данным можно получить доступ разными путями (через глобальную переменную и стандартную конфигурацию), что может запутывать при разработке и чтении кода. Давай избавимся от глобальных переменных, которые присутствуют в стандартной структуре

File: ImageFeedTests/WebViewTests.swift [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    let authHelper = AuthHelper()
+                    
+                    //when
+                    let code = authHelper.code(from: url)
+                    
+                    //then
+                    XCTAssertEqual(code, "test code")
+                }
+                
+            }
```
Comment (DanilaG) [inline ImageFeedTests/WebViewTests.swift:93]: Давай поправим отступы

File: ImageFeedUITests/ImageFeedUITests.swift
Comment (nnkmk) [inline ImageFeedUITests/ImageFeedUITests.swift:1]: 🌝

Comment (nnkmk) [reply ImageFeedUITests/ImageFeedUITests.swift:1]: Поправил, вроде как работает. Я устал от этого проекта, буду благодарен за понимание.
